demo/app.component.ts

```diff
@@ -107,6 +108,25 @@ export class AppComponent implements OnInit {
     this.form2 = this.formBuilder.group({ 'range': [ [ 2, 8 ] ] });
   }
 
+  public onSliderInit(slider: any) {
+    this.sliderObject  = slider;
+    console.debug('Slider Object Instance Initiated and available for use in your component');
+    console.debug(this.sliderObject);
+  }
+
+  public resetSlider() {
+    this.sliderObject.reset();
+  }
+
+  public togglePips() {
```

Comment:
Yes you are right. I now see that we can access the slider object from the elementRef. That is cool. But the provblem is also that there is no event that says in the moment. Hey I'm initialized and ready to go. So what happens is now is I have to go with <code>if(!this.mySliderObject) { this.mySldierObject = this.myElementRef.slider}</code>constantly on every ngCheck() or ngAfterContentChecked(). Did I miss some event in which you can be certain the the slider object is available?